content/en/docs/reference/command-line-tools-reference/feature-gates.md

```diff
@@ -100,7 +100,7 @@ different Kubernetes components.
 | `ExpandPersistentVolumes` | `true` | Beta | 1.11 | |
 | `ExperimentalHostUserNamespaceDefaulting` | `false` | Beta | 1.5 | |
 | `EvenPodsSpread` | `false` | Alpha | 1.16 | 1.17 |
-| `EvenPodsSpread` | `true` | Beta | 1.18 | |
+| `EvenPodsSpread` | `true` | Beta | 1.18 | 1.18 |
```

Comment:
I'm afraid you need to do this differently. Remove the feature gate from _Feature gates for Alpha or Beta features_ and move it to the table for stable and graduated features.

Reply:
Ah, definitely. Updated.